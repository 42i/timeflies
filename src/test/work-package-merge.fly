wp aa
    bbb
        xxx
        yyy
    ccc
    ddd

wp aa.bbb.xxx
    sss
    ttt
    uuu
    
wp aa.ccc
    gogo
    gugu
    gaga