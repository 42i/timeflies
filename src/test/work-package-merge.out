Work package breakdown:
ALL
    aa
        bbb
            xxx
                sss
                ttt
                uuu
            yyy
        ccc
            gogo
            gugu
            gaga
        ddd
